drivers.toml

```diff
@@ -0,0 +1,18 @@
+# If adding a new driver, please try to keep the entries alphabetized! :)
+
+[drivers.drv8825]
+name = "drv8825"
```

Comment:
Isn't that redundant? You get the driver name from the map key, as well as the struct. Probably best to choose the one that is most useful in the code and remove the other.

So either:
``` toml
[drivers.drv8825]
# no `name`
# ...
```

Or:
``` toml
[[drivers]]
name = "drv8825"
# ...
```